content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -5,48 +5,146 @@ description: >
   Learn how to install the Armory Agent in your Kubernetes and Armory Enterprise environments.
 weight: 2
 ---
-![Proprietary](/images/proprietary.svg)
+![Proprietary](/images/proprietary.svg) This feature requires an Armory licensed entitlement.
 
-## Compatibility matrix
+## Overview
 
-{{< include "agent/agent-compat-matrix.md" >}}
-
-The Agent consists of a service deployed as a Kubernetes `Deployment` and a plugin to Spinnaker's Clouddriver service. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
+The Agent consists of a plugin to Spinnaker's Clouddriver service and a K8s deployment that connects to Clouddriver. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
 
 ## {{% heading "prereq" %}}
 
-* This guide is for experienced Kubernetes and Armory Enterprise users.
+* This guide assumes you're using the Armory Operator to install Spinnaker, with the Kustomize method from the [spinnaker-kustomize-patch repo](https://github.com/armory/spinnaker-kustomize-patches).
+
 * You have read the Armory Agent [overview]({{< ref "armory-agent" >}}).
-* You have a Redis instance. The Agent uses Redis to coordinate between Clouddriver replicas.
 * You have configured Clouddriver to use MySQL or PostgreSQL. See the {{< linkWithTitle "clouddriver-sql-configure.md" >}} guide for instructions.
+* You have Spinnaker installed in one cluster, and an additional cluster to serve as your target deployment cluster.
+
+### Compatibility matrix
+
+{{< include "agent/agent-compat-matrix.md" >}}
+
+### Networking requirements
 
-## Networking requirements
+Communication from the Agent to Clouddriver occurs over gRPC port 9091.
```

Comment:
I like that we're spelling out gRPC and "from Agent to clouddriver", but I'd leave the http/2 comment. Some people are not familiar with it enough to understand the implication that it won't work with any ingress or proxy they are already using (for the out of cluster case).

Reply:
Ok. Something I just discovered (had flown under my radar all these years) is most of the mainstream ingress controllers can layer 7 grpc. I set this up with L4 LB expose.